src/helpers/array.js

```diff
@@ -0,0 +1,3 @@
+export function intersect (array1 = [], array2 = []) {
+  return array1.filter(value => array2.indexOf(value) !== -1).filter((e, i, c) => c.indexOf(e) === i);
```

Comment:
maybe explicit what are `e`, `i` and `c` with more verbose names?

Reply:
Good one. I like to live on the edge a bit too much.